bin/connect-distributed.sh

```diff
@@ -26,6 +26,10 @@ if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
     export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/connect-log4j.properties"
 fi
 
+if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
+    export KAFKA_HEAP_OPTS="-Xmx1G -Xms1G"
```

Comment:
@ewencp perhaps we should make it `-Xmx1G -Xms256M`?

Or, maybe 512M as default max is a more conservative bump for now?